tests/src/commonMain/kotlin/kotlinx/document/database/tests/AbstractCacheOverflowTests.kt

```diff
@@ -0,0 +1,52 @@
+package kotlinx.document.database.tests
+
+import kotlinx.document.database.DataStore
+import kotlinx.document.database.getObjectCollection
+import kotlin.js.JsName
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+import kotlin.time.Duration.Companion.days
+
+abstract class AbstractCacheOverflowTests(val store: DataStore) : BaseTest(store) {
+    companion object {
+        val commitInterval = 1.days
+    }
+
+    abstract fun getUnsavedMemory(): Int
+
+    abstract fun getTotalCacheMemorySize(): Int
+
+    @Test
+    @JsName("cache_is_flushed_when_memory_is_full")
+    fun `test if Cache Is flushed when memory is full`() =
+        runDatabaseTest {
+            assertTrue(
+                actual = store.configuration.commitPolicy is DataStore.CommitPolicy.Periodic,
+                message = "Commit policy should be periodic",
+            )
+            assertEquals(
+                expected = 1.days,
```

Comment:
`assert(store.configuration.commitPolicy as DataStore.CommitPolicy.Periodic).interval >= 1.days) { "error" }` is more appropriate.